server/entity/src/user.rs

```diff
@@ -1,15 +1,20 @@
+use crate::Id;
 use sea_orm::entity::prelude::*;
 use serde::{self, Deserialize, Serialize};
 
 #[derive(Clone, Debug, PartialEq, DeriveEntityModel, Deserialize, Serialize)]
 #[sea_orm(table_name = "users")]
 pub struct Model {
     #[sea_orm(primary_key)]
-    pub id: i32,
+    pub id: Id,
+    #[sea_orm(unique)]
     pub email: String,
     pub username: String,
     #[serde(skip_serializing)]
     pub pw_hash: String,
+    #[sea_orm(default_value = "false")]
+    #[serde(skip_serializing)]
+    pub activated: bool,
```

Comment:
Do you mind adding this to the ER model? 😬

Reply:
No need for the emoji, plantuml is easy to modify.